spec/features/admin/business_units_spec.rb

```diff
@@ -14,14 +15,12 @@
     expect(business_group_row).to be_present
     business_group_row.name.click
     expect(teams_show_page.heading)
-      .to have_text "You are viewing Business group #{bu.business_group.name}"
-
+      .to have_copy "You are viewing Business group #{bu.business_group.name}"
```

Comment:
Doesn't adding `visible: true` as an argument to `have_copy` work?

Reply:
I'll give this a try.

Reply:
I tried  this 
```
expect(teams_show_page.heading)
      .to have_text("You are viewing Business group #{bu.business_group.name}",visible: true)
```

and got this error:
```
1) editing teams editing a business unit
     Failure/Error:
       expect(teams_show_page.heading)
         .to have_text("You are viewing Business group #{bu.business_group.name}",visible: true)
     
     ArgumentError:
       invalid keys :visible, should be one of :count, :minimum, :maximum, :between, :wait, :exact
     # ./spec/features/admin/business_units_spec.rb:17:in `block (2 levels) in <top (required)>'

```

Reply:
This commit introduces the new have_copy matcher: 2c553267f79d351cad5f1efaf6f17f0297f3341c to address the issue with hidden text

Reply:
Sorry I got that wrong in my comment. What I should've said is you can add `visible: false` to the element matcher in the SitePrism object, for example in [PageObjects::Sections::Cases::TypeFilterPanelSection](https://github.com/ministryofjustice/correspondence_tool_staff/blob/master/spec/site_prism/page_objects/sections/cases/type_filter_panel_section.rb#L8). This is how I've handled this in the past.

Reply:
Ok, this is slowly coming into focus now. :) So Chrome's web driver is matching differently now, not finding hidden text, and using `visible: false` on the matcher is only useful when the entire element is hidden. I think I'm up-to-speed now.